docs/painless/painless-contexts/painless-watcher-condition-context.asciidoc

```diff
@@ -1,38 +1,146 @@
 [[painless-watcher-condition-context]]
 === Watcher condition context
 
+include::painless-watcher-context-variables.asciidoc[]
+
+==== Watch condition
```

Comment:
I'd remove this and the Action condition heading and put both examples under an "Example" heading/label. (Like API, Return, etc.)